extension/package.json

```diff
@@ -1319,6 +1337,11 @@
           "name": "Actions",
           "when": "true"
         },
+        {
+          "id": "dvc.views.studio",
+          "name": "Studio",
+          "when": "!dvc.studio.connected"
```

Comment:
[F] We only show the tree view when there is no Studio token available in the secrets store. It can still be hidden by the user. We also give them the option to remove the token and save another one.